src/Stryker.Core/Stryker.Core/Initialisation/InputFileResolver.cs

```diff
@@ -157,7 +174,7 @@ public string ScanProjectFile(string currentDirectory)
                     sb.AppendLine(file);
                 }
                 sb.AppendLine();
-                sb.AppendLine("Please fix your project contents");
+                sb.AppendLine("Please specify a testProjectFile.");
```

Comment:
You could add examples here as we do in find project under test since we already know all projectFiles the user could choose.